server/src/service/ReviewsService.js

```diff
@@ -0,0 +1,46 @@
+const reviewsDao = require('../repository/ReviewsDAO');
+const uuid = require('uuid');
+
+async function createNewReview(receivedData) {
+    //validate data before posting a new review
+    if(validateFields(receivedData)) {
+        //get the created at time
+        const time = Date.now();
+
+        //create new review
+        const review = {
+            recipeId: uuid.v4(),
+            reviewId: uuid.v4(),
+            author: receivedData.username,
+            imageUrl: receivedData.imageUrl,
+            rating: receivedData.rating,
+            content: receivedData.content,
+            createdAt: time
+        }
+        //create the new post
+        const data = await reviewsDao.postReview(review);
+        if(data) {
+            return review;
+        } else {
+            return false;
+        }
+    }
+    else {
+        return 'Please fill out fields correctly';
+    }
+}
+
+
+function validateFields(data) {
```

Comment:
Validation needs to be more encompassing.  A client can give a String for rating, for instance.  For rating, check for max and min values.  There may be more checks needed.